recap-loops/homework-solutions.md

```diff
@@ -0,0 +1,371 @@
+---
+title: "Assignment Solutions"
+nav_order: 1
```

Comment:
this should be 2 I think. Please feel free top merge. Looks good.

Reply:
@olayinkasf : ah, good point thanks, will change!